### ISSUE TITLE
style

### ISSUE THREAD

**Issue by tarekziade** [faitmain/fablab.faitmain.org#3]
rendre la page sexy

**Comment by raphaelbastide** [comment]
- [ ] Intégrer les fontes de faitmain.org
- [ ] Reprendre les couleurs de faitmain.org

**Comment by tarekziade** [comment]
- [x] brancher http://maps.stamen.com/toner/#12/37.7073/-121.9755

**Comment by tarekziade** [comment]
- [ ] rendre le design responsive

**Comment by raphaelbastide** [comment]
Oui, dommage pour les petites résolutions avec le calendrier. De plus je trouve que cette texture de map vide de texte n'est pas très pratique, malgré le fait qu'elle soit cool :)

**Comment by tarekziade** [comment]
@raphaelbastide  je suis à l'apogée de mes capacités en design et CSS ^^
